### ISSUE TITLE
Ctrl+Tab as a shortcut

### ISSUE THREAD

**Issue by ilyachase** [orschiro/switch-recent-tabs-chrome-extension#35]
Hi there. Extension works like a charm and it is what I wanted.
But only problem is, that Google Chrome seems to not allow Ctrl+Tab as a shortcut option. It changes tab from 'settings' to another one.
In older version of Chrome I remember that I changed shortcut in config file and then changed permissions so that Chrome could not reassign my hotkey and Ctrl+Tab worked. In new version (65.0.3325.181) I couldn't do that. Can you guys help?

Basically issues sounds like that: "Can I somehow assign **any** key for your extension?".

**Comment by orschiro** [comment]
Hey @ilyachase 

> In older version of Chrome I remember that I changed shortcut in config file and then changed permissions so that Chrome could not reassign my hotkey and Ctrl+Tab worked. In new version (65.0.3325.181) I couldn't do that. Can you guys help?

I am not sure but I believe Ctrl + Tab is reserved and blocked by Chrome for developers to use it. Have you already raised your request at Chromium upstream?

**Comment by orschiro** [comment]
Status: Won't fix. I am afraid there is nothing we can do for now. :-(
